arcgis-ios-sdk-samples/Layers/Group layers/LayersTableViewController.swift

```diff
@@ -97,32 +127,49 @@ class LayersTableViewController: UITableViewController, GroupLayersCellDelegate,
     /// - Parameters:
     ///     - section: Section of the table view.
     ///     - enabled: Indicates if the switch should be enabled or disabled.
-    func updateSwitchForRows(in section: Int, isEnabled: Bool) {
+    func updateCells(in section: Int, isEnabled: Bool) {
         guard let visibleRows = tableView.indexPathsForVisibleRows else { return }
-        
-        visibleRows.lazy.filter { $0.section == section }.forEach { indexPath in
-            if let cell = tableView.cellForRow(at: indexPath) as? GroupLayersCell {
-                cell.layerVisibilitySwitch.isEnabled = isEnabled
+        switch section {
+        case 0:
+            visibleRows.lazy.filter { $0.section == section }.forEach { indexPath in
+                if let cell = tableView.cellForRow(at: indexPath) as? GroupLayersCell {
+                    cell.isUserInteractionEnabled = isEnabled
+                    cell.layerVisibilitySwitch.isEnabled = isEnabled
+                }
+            }
+        case 1:
+            visibleRows.lazy.filter { $0.section == section }.forEach { indexPath in
+                if let cell = tableView.cellForRow(at: indexPath) {
+                    cell.isUserInteractionEnabled = isEnabled
+                    cell.textLabel?.isEnabled = isEnabled
+                    // Change the tint if the cell is enabled.
+                    if isEnabled {
+                        cell.tintColor = .none
+                    } else {
+                        cell.tintColor = .gray
+                    }
```

Comment:
With `isUserInteractionEnabled` set, I believe the tint color are managed automatically. Please try to comment out and see if it behaves as expected. 😳

Reply:
I had to include this specifically because the accessory checkmarks for those cells didn't change colors despite changing `isUserInteractionEnabled`.

Reply:
Good point!